SportHub.API/appsettings.json

```diff
@@ -5,5 +5,12 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "ConnectionStrings": {
+    "DefaultConnection": "server=localhost;database=test;user=root;password=rootadmin2022"
```

Comment:
Explain to other team members how the need to modify this default connection if they want to use your api